src/transformers/__init__.py

```diff
@@ -442,12 +442,14 @@
         DataCollator,
         DataCollatorForLanguageModeling,
         DataCollatorForPermutationLanguageModeling,
+        DataCollatorForNSP,
```

Comment:
`DataCollatorForNextSentencePrediction` is consistent with our naming patttern

Reply:
Received, thanks for the reply.
The work I am doing recently involves  pre-training tasks using sentence-level only. Therefore, I write `DataCollatorForNSP` separately.
So do you suggest modifying directly on `DataCollatorForLanguageModeling` to add the codes for the NSP task? 
Is it necessary to keep the separate `DataCollatorForNSP`?

Reply:
I will commit the name modification later. `DataCollatorForNSP' -> `DataCollatorForNextSentencePrediction'

Reply:
> So do you suggest modifying directly on `DataCollatorForLanguageModeling` to add the codes for the NSP task?
> Is it necessary to keep the separate `DataCollatorForNSP`?

No I'm not suggesting that! I only wonder if a separate `run_bert_nsp.py` (without MLM) script makes sense. What do you think? @sgugger @sshleifer